tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+                </div>
+            </foreignObject>
+        </svg>
+
+        <p>
+            Now that we have a better understanding of what <code>foreignObject</code> is all about, how would we create 
+            an equivalent example with JointJS? Using <code>foreignObject</code> with JointJS is much the same as creating any 
+            <a href="/tutorial/custom-elements" target="_blank"><code>custom element</code></a> which you can see in the following
+            code example.
+        </p>
+
+        <pre><code>const namespace = joint.shapes;
+
+const graph = new joint.dia.Graph({}, { cellNamespace: namespace });
+
+const paper = new joint.dia.Paper({
+    // paper options
+    guard: (evt) => ['P'].includes(evt.target.tagName)
+});
+
+const Card = joint.dia.Element.define('example.ForeignObject', {
+    attrs: {
+        body: {
+            width: 'calc(w)',
+            height: 'calc(h)',
+            fill: {
+                type: 'linearGradient',
+                stops: [
+                    { offset: 0, color: '#ff5c69' },
+                    { offset: 0.5, color: '#ff4252' },
+                    { offset: 1, color: '#ed2637' }
+                ]
+            }
+        },
+        foreignObject: {
+            width: 'calc(w-12)',
+            height: 'calc(h-12)',
+            x: 6,
+            y: 6
+        }
+    },
+}, {
+    markup: [
+        {
+            tagName: 'rect',
+            selector: 'body'
+        },
+        {
+            tagName: 'foreignObject',
+            selector: 'foreignObject',
+            children: [
+                {
+                    tagName: 'div',
+                    namespaceURI: 'http://www.w3.org/1999/xhtml',
+                    selector: 'background',
+                    style: {
+                        backgroundColor: '#131e29',
+                        height: '100%'
+                    },
+                    children: [
+                        {
+                            tagName: 'p',
+                            selector: 'text',
+                            style: {
+                                color: '#F2F2F2',
+                                font: '16px sans-serif',
+                                padding: '10px',
+                                margin: 0,
+                            },
+                            textContent: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+});
+
+const card = new Card({
+    position: { x: 10, y: 10 },
+    attrs: {
+        body: { width: 180, height: 100 }
+    }
+});
+
+card.addTo(graph);
+</code></pre>
+
+        <p>
+            As mentioned earlier, we would like to create an equivalent example with JointJS, that means selecting text needs
+            to be possible. In order to achieve this functionality, we need to use the <code>paper</code>
```

Comment:
the first sentence should be rewritten - something like "In order for our JointJS example to have equivalent functionality as the first example, we need to make selecting text possible. To do that, we use the paper guard option..."